### DISCUSSION TITLE
Plans for v1 - Want to contribute

### DISCUSSION THREAD

**Discussion by angcl** [PDMLab/docker-compose#265] [General]
Hi there,

I would love to contribute to this project and would like to know if you kinda have a plan of what's missing for the initial v1 release.

Best regards
Angelo

**Comment by AlexZeitler** [comment]
Hi @angcl,

thanks for your interest in contributing.

What's actually holding me back is lack of complete docs.

Thanks,

Alex

**Reply by angcl** [reply]
Hi @AlexZeitler,

that's a good goal to aim at!

Thanks
Angelo

**Comment by AlexZeitler** [comment]
Current version
![image](https://github.com/PDMLab/docker-compose/assets/287480/56dce5b7-b7ec-4fab-87ce-90d270d26682)


Instead of the single page for all commands I would strive for a page per command and I aim for different docs for v1 / v2

So something like

- v1
  - ps
  - up
  - down
- v2
  - ps
  - up
  - down

Reason is we have `downOne` `downMany` and `downAll` for v2 but not for v1 because the v1 binary doesn't support it.

**Reply by angcl** [reply]
I will contribute to the documentation from time to time. What do you think about adding JSDoc as well?

**Reply by AlexZeitler** [reply]
Thanks. I think JSDoc could make sense but it needs to be maintained as well.
